docs/contributing/pullrequest-submission-and-lifecycle.md

```diff
@@ -219,6 +219,8 @@ resource/aws_example_thing: Add `not_broken` attribute
 
 The CHANGELOG is intended to show operator-impacting changes to the codebase for a particular version. If every change or commit to the code resulted in an entry, the CHANGELOG would become less useful for operators. The lists below are general guidelines and examples for when a decision needs to be made to decide whether a change should have an entry.
 
+Pull requests that do not require a changelog should be marked with the `pr/no-changelog` label. This exempts them from the PR checking automation.
```

Comment:
I do not believe contributors will be able to do this themselves, so the PR check will fail until a maintainer does it or the contributor (potentially extraneously) adds a file. I'm not sure if we want to mention that here as something that they should ask for or that the maintainers will ignore the failed check if the changelog entry is unnecessary.